README.md

```diff
@@ -30,3 +30,13 @@ Once the values match your environment run the following.
 ```bash
 kubectl create -n default -f ifps.yaml
 ```
+
+### Running in KIND
+
+An easy way to test and modify changes to the operator is by running it in a local KIND cluster.
+To bootstrap a KIND cluster, you can run `hack/setup-kind-cluster.sh`, which will install all of the 
+required components to operate an IPFS cluster. 
+
+To deploy the operator in this repository into the cluster, you can run `hack/run-in-kind.sh` which
+will build the source code and inject it into the cluster.
+If you make subsequent changes, you will need to re-run `hack/run-in-kind.sh` in order to redeploy local changes. 
```

Comment:
Do we want to provide steps on how to test changes using the kuttl tests locally?